src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprints.java

```diff
@@ -23,6 +23,15 @@ private static String trim(String imageId) {
         }
         return imageId.substring(0, 32);
     }
+    
+    /**
+     * Gets a fingerprint hash for Docker image.
+     * @param imageId Docker image ID.
+     * @return 32-symbol fingerprint hash
+     */
+    /*package*/ static @Nonnull String getImageFingerprintHash(@Nonnull String imageId) {
+        return trim(imageId);
```

Comment:
I do not care much about the name; anyway the proposed new method is not `public` either. Just commenting that it is odd to have _both_ methods.